index.html

```diff
@@ -0,0 +1,235 @@
+                        <p>I'm a software developer! I can help you build a product,
+                            feature or website. Look through someof my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <nav id="lets-connect">
+                        <h5>LET'S CONNECT</h5>
+                    </nav>
+                    <div>
+                        <ul class="icons">
+                            <li><img src="Vector.png" alt=""></li>
+                            <li><img src="Linkedin.png" alt=""></li>
+                            <li><img src="Vector1.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                        </ul>
+                    </div>
+                    <section class="bottom-images">
+                        <div id="elipse12">
+                            <img src="Ellipse12.png" alt="">
+                        </div>
+                        <div id="elipse13">
+                            <img src="Ellipse13.png" alt="">
+                        </div>
+                        <div id="elipse15">
+                            <img src="Ellipse15.png" alt="">
+                        </div>
+                        <div id="elipse4">
+                            <img src="Ellipse4.png" alt="">
+                        </div>
+                        <div id="rectangle5">
+                            <img src="Rectangle5.png" alt="">
+                        </div>
+                    </section>
+                </section>
+            </div>
+    </article>
+        
+        <section class="work-card">
+            <div class="work1">
+                <img src="Snapshoot.png" alt="">
+               
+                <h2>Tonic</h2>
+                <div id="frame">
+                    <div id="text1"><h5>CANOPY</h5></div>
+                    <div id="counter"></div>
+                    <div id="role"><h5>Back End Dev</h5></div>
+                    <div id="counter1"></div>
+                    <div id="text"><h5>2015</h5></div>
+                </div>
+                <div class="primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="tags">
+                    <div id="tag1"><div id="html">html</div></div>
+                    <div id="tag2"><div id="css">css</div></div>
+                    <div id="tag3"><div id="javascript">javascript</div></div>
+                </div>
+                <div class="Actions">
+                    <div id="see-project">
+                        <div><h4>See Project</h4></div>
+                    </div>
+                </div>
+            </div>
+            <div class="work2">
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you align your project contents properly

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/177016402-616369a7-e853-40c0-974c-7ed3a0278121.png) | ![image2](https://user-images.githubusercontent.com/52934128/177016421-6f8d5076-89c8-429e-8831-addcce0191b6.png) |